app/models/admin_menu.rb

```diff
@@ -0,0 +1,76 @@
+class AdminMenu
+  # On second level navigation with more children, we reference the default tabs controller. i.e look at developer_tools
+  # rubocop:disable Metrics/BlockLength
+  ITEMS = Menu.define do
+    scope :people, [
+      item(name: "people", controller: "users"),
+    ]
+
+    scope :content_manager, [
+      item(name: "posts", controller: "articles"),
+      item(name: "badges", children: [
+             item(name: "badge library", controller: "badges"),
+             item(name: "badge achievements"),
+           ]),
+      item(name: "organizations"),
+      item(name: "podcasts"),
+      item(name: "tags"),
+    ]
+
+    scope :customization, [
+      item(name: "config"),
+      item(name: "HTML variants", controller: "html_variants"),
+      item(name: "display_ads"),
```

Comment:
I see that sometimes you use `_` and other times you use the space to separate words, relying on Rails casing abilities. Maybe we can uniform that to spaces, feels more naturale as it's the "human" description for the section

Reply:
This was a completely unintentional copy paste error from the controller names. Thanks for picking this up, they should be spaced and not underscored. ✅